source/routing/raptor_solution_reader.cpp

```diff
@@ -727,8 +733,22 @@ Path make_path(const Journey& journey, const type::Data& data) {
 
         //we then need to create one section by service extension
         const VehicleSection* last_vj_section = nullptr;
-        for (const auto& vj_section: get_vjs(section)) {
-            if (last_vj_section) {
+        const auto& sections = get_vjs(section);
+        for (size_t i(0); i < sections.size(); i++) {
+            auto& vj_section = sections.at(i);
+            const auto& boarding_st = vj_section.stop_times_and_dt.front().st;
+            const auto& alighting_st = vj_section.stop_times_and_dt.back().st;
+            if (i == 0 && boarding_st.boarding_time != boarding_st.departure_time) {
+                // Add a boarding section if we have boarding_time != departure_time and that's the first vj
+                path.items.emplace_back(ItemType::boarding,
+                                        posix(section.get_in_dt),
+                                        posix(section.get_in_dt + boarding_st.get_boarding_duration()));
+                auto& boarding_section = path.items.back();
+                boarding_section.stop_points.push_back(boarding_st.stop_point);
+                boarding_section.stop_points.push_back(boarding_st.stop_point);
```

Comment:
@TeXitoi explained to me why L747 is the same as L748... just for the clarity, can you add a comment plz? :)